sdk/core/Azure.Core/src/Response.cs

```diff
@@ -31,5 +31,9 @@ public abstract class Response : IDisposable
 
         protected internal abstract IEnumerable<HttpHeader> EnumerateHeaders();
 
+        public Response<T> WithValue<T>(T value)
```

Comment:
Nit - Do we want a corresponding `FromValue` static helper on `Response<T>` for discoverability?  I generally look past `With` methods as conveniences.  We can always add it in the future too, but I guess i'd expect `Response<T>.FromValue` to be the first thing we offer instead of `Response.WithValue`.

Reply:
This is why I didn't do it: https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1000-do-not-declare-static-members-on-generic-types?view=vs-2019

Reply:
Oh, yeah....  I guess it’d have to be `Response.FromValue` then.  I’m fine with it as-is.

Reply:
I talked with Pavel, he will change it to Response.FromValue<T>(...)

Reply:
@KrzysztofCwalina seems to also like `Response.FromValue` so that's that.